services/dashboard/views/apps/compose/popups/choose_edge.html

```diff
@@ -1,5 +1,5 @@
 <div class="modal" tabindex="-1">
-  <div class="modal-dialog modal-md">
+  <div class="modal-md col-md-5 col-md-offset-3">
```

Comment:
`col-md-5 col-md-offset-3` Do you want to a center alignment? Please try `modal-md text-center`